uyuni/common-libs/uyuni-common-libs.spec

```diff
@@ -108,14 +108,16 @@ cp $RPM_BUILD_ROOT%{python3root}/common/{checksum.py,cli.py,rhn_deb.py,rhn_mpm.p
     $RPM_BUILD_ROOT%{python2root}/common
 %endif
 
+%if 0%{?suse_version}
+%if 0%{?build_py2}
```

Comment:
This macro is not defined here in the spec. Either we define it and skip also building the python2-* package or we remove it.

Reply:
I guess you mean `build_py2` for CentOS/RHEL/RES?

That's right, but it's inside `%if 0%{?suse_version}`, so do we really need to define it? Package is building fine for me.

Reply:
"Never called" for SLE becase `build_py2` is not defined. Got it now, I think.